cdap-docs/reference-manual/source/cli-api.rst

```diff
@@ -184,5 +184,22 @@ These are the available commands:
    ``send stream <stream-id> <stream-event>``,"Sends an event to a Stream."
    **Egress**
    ``call procedure <app-id.procedure-id> <app-id.method-id> [<parameter-map>]``,"Calls a Procedure."
-   ``call service <app-id.service-id> <http-method> <endpoint> [headers <headers>] [body <body>]``,"Calls a Service endpoint. The <headers> are formatted as ""{'key':'value', ...}"" and the <body> is a String." 
+   ``call service <app-id.service-id> <http-method> <endpoint> [headers <headers>] [body <body>]``,"Calls a Service endpoint. The <headers> are formatted as ""{'key':'value', ...}"" and the <body> is a String."
+   **Preferences**
+   ``get instance preferences``, "Get Preferences of CDAP Instance."
+   ``get namespace preferences``, "Get Preferences of the Namespace."
+   ``get app preferences <app-id>``, "Get Preferences of a Application."
+   ``get flow preferences <app-id.flow-id>``, "Get Preferences of a Flow."
+   ``get namespace resolved preferences``, "Get Resolved Preferences of the Namespace."
+   ``get app resolved preferences <app-id>``, "Get Resolved Preferences of a Application."
+   ``get flow resolved preferences <app-id.flow-id>``, "Get Resolved Preferences of a Flow."
+   ``set instance preferences <arguments>``, "Set Preferences of CDAP Instance. <arguments> is specified in the format ""key1=a, key2=b"".""
```

Comment:
These next lines look incorrect, and are probably what are causing the table not be formatted correctly. They should have a single double-quote at the end, after the period, neither two nor three:

```
""key1=a, key2=b""."
```